content/quickstart/kubernetes/tutorial-eks.md

```diff
@@ -2,10 +2,11 @@
 title: "Tutorial: EKS - Hello World!"
 aliases:
     - tutorial-eks.html
-    - /quickstart/eks-hello-world.html
+    - /quickstart/kubernetes/eks-hello-world.html
```

Comment:
```suggestion
    - /quickstart/aws/tutorial-eks/
    - /quickstart/eks-hello-world.html
```

The `/quickstart/eks-hello-world.html` alias should remain. And one should be added for the old pretty-style URL location: `/quickstart/aws/tutorial-eks/`.

Altogether the aliases for this file should be:

```
aliases:
    - /quickstart/eks-hello-world.html
    - /quickstart/aws/tutorial-eks.html
    - /quickstart/aws/tutorial-eks/
```